Gulpfile.coffee

```diff
@@ -190,5 +212,34 @@ gulp.task 'serve', ['watch'], ->
                 res.setHeader('Content-Type','image/svg+xml');
             next()
     ]
-
   connect.server(config)
+
+gulp.task '_tdd', (done) ->
+  buildAndTest(true, done)
+  return # Since this is async
+
+
+# -----------------------------------------------------------------------
+#  Public Tasks
```

Comment:
I like the second idea, but yeah, that just make prod-tasks a little more confusing...or maybe prod/_archive can live in gulpfile as well?